ibm_catalog.json

```diff
@@ -61,6 +61,11 @@
               }
             ]
           }
+        },
+        {
+          "label": "Private Only",
+          "name": "private",
+          "working_directory": "examples/private"
```

Comment:
you need to also include and `architecture` section. Just copy same format as basic example